client/views/room/contextualBar/UserInfo/UserInfo.js

```diff
@@ -34,6 +35,18 @@ function UserInfo({
 
 	const timeAgo = useTimeAgo();
 
+	const customFieldsToShowSetting = useSetting('Accounts_CustomFieldsToShowInUserInfo');
+	const customFieldsToShow = [];
+	try {
```

Comment:
@yash-rajpal we should replace this `try-catch` and use an if statement

Reply:
Thanks for the review. Done the required Changes. :)
This is my config of custom fields -
![image](https://user-images.githubusercontent.com/58601732/117813759-1bb05b80-b281-11eb-8f08-7d674fca82d0.png)
custom fields to show is 
![image](https://user-images.githubusercontent.com/58601732/117813831-31258580-b281-11eb-9650-54ace734df7a.png)

I have also included a Check that if value of any custom field is missing, then we should not render that field, as that would be empty.
![image](https://user-images.githubusercontent.com/58601732/117814019-67630500-b281-11eb-8723-4200aaab28a7.png)
Label will be rendered as given in json, so showing `roleTest` label inplace of `role`.

![image](https://user-images.githubusercontent.com/58601732/117814133-8f526880-b281-11eb-92df-bd6b4a592f1e.png)
Here `Test` custom field is missing so not rendering that.

Reply:
@yash-rajpal I'll check it